hkmc2/shared/src/main/scala/hkmc2/semantics/Importer.scala

```diff
@@ -60,15 +60,9 @@ class Importer:
           val res = p.parseAll(p.block(allowNewlines = true))
           val resBlk = new syntax.Tree.Block(res)
           
-          // * Note: we don't even need to elaborate the block!
-          // * Though doing so may be needed later for type checking,
-          // * so we should probably do it lazily in the future.
-          /* 
-          given Elaborator.State = new Elaborator.State
-          given Elaborator.Ctx = Elaborator.Ctx.init.nest(N)
-          val elab = Elaborator(tl, file / os.up)
-          val (blk, newCtx) = elab.importFrom(resBlk)
-          */
+          given Elaborator.Ctx = self.prelude.copy(mode = Mode.Light).nest(N)
```

Comment:
Why use `self.prelude` here and just `prelude` below?
```suggestion
          given Elaborator.Ctx = prelude.copy(mode = Mode.Light).nest(N)
```